### PR TITLE
Built the Checkin modal component

### PR DESCRIPTION
**Pull request by Holamite**
## Checkin modal component - FIXES: https://github.com/Explore-Beyond-Innovations/ZeroCheck_Frontend/issues/5
- I created the CheckIn file in the component folder
- I install react QR code 
- I made the timer countdown functional 
- I made the UI responsive

### CODE REVIEW

File: app/check-in/page.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React, { useState } from "react";
+import CheckInModal from "../components/Check-In";
```
Comment (Ugo-X) [inline app/check-in/page.tsx:2]: theres no use client hook as you are importing usestate in this file, so it doesnt compile. fix

**Review by Ugo-X — CHANGES_REQUESTED** [review]

File: app/check-in/page.tsx (hunk 1 above)
Comment (Holamite) [reply app/check-in/page.tsx:2]: Alright, I'm on it!

**Comment by Holamite** [comment]
I have made the require fix to make it compile.

**Review by Ugo-X — APPROVED** [review]
changes approved

**Pull request merged**